src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -51,64 +51,64 @@
     <Compile Include="$(CommonTestPath)\System\Reflection\MockParameterInfo.cs">
       <Link>Common\System\Reflection\MockParameterInfo.cs</Link>
     </Compile>
+    <Compile Include="TimeZoneInfoTests.cs" />
```

Comment:
Not yet as this would've added another 500 lines to the diff. The file that did move was TimeZoneTests :)